common/src/main/resources/data/cornexpansion/recipes/cheesy_popcorn.json

```diff
@@ -0,0 +1,21 @@
+{
+    "type": "minecraft:crafting_shapeless",
+    "ingredients": [
+      {
+        "item": "cornexpansion:popcorn"
+      },
+      {
+        "item": "cornexpansion:popcorn"
+      },
+      {
+        "item": "cornexpansion:popcorn"
+      },
+      {
+        "tag": "c:cheese"
```

Comment:
Recipes using tags should be put inside their respective folder, ie c: in fabric and forge: in forge

Reply:
fixed!

Reply:
So question - mostly for my understanding around this same topic. Should the popcorn and corn_on_the_cob recipes be under a farm_and_charm folder instead of cornexpansion?

Reply:
the recipe with the c: tag should be in `/fabric/src/main/resources/data/cornexpansion/recipes/cheesy_popcorn.json` and the one with the forge tag in `/forge/src/main/resources/data/cornexpansion/recipes/cheesy_popcorn.json`

The other recipes are fine, recipes should be in the farm_and_charm folder if they only use items from F&C to make a F&C item, or if adding compat